web/src/connectors/task-annotator-connector/task-annotator-context.tsx

```diff
@@ -223,6 +254,9 @@ export const TaskAnnotatorContextProvider: React.FC<ProviderProps> = ({
     >({});
     const [externalViewer, setExternalViewer] =
         useState<ExternalViewerState>(defaultExternalViewer);
+    const [selectedRelatedDoc, setSelectedRelatedDoc] = useState<FileDocument | undefined>(
```

Comment:
This was extracted from "useDocumentLinks" custom hook forcedly until we split the context into different parts.